.github/stale.yml

```diff
@@ -1,25 +1,26 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 210
+daysUntilStale: 30
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 15
+daysUntilClose: 60
 # Issues with these labels will never be considered stale
 exemptLabels:
   - pinned
   - security
+  - assigned
```

Comment:
Instead of using a new label that will need to be assigned manually, why we don't use `exemptAssignees: true` ?